test/modules/test_sql_injection_detector.py

```diff
@@ -0,0 +1,102 @@
+# type: ignore
+from test.base import ArtemisModuleTestCase
+
+from karton.core import Task
+
+from artemis import http_requests
+from artemis.binds import Service, TaskStatus, TaskType
+from artemis.modules.sql_injection_detector import SqlInjectionDetector
+
+
+class PostgresSqlInjectionDetectorTestCase(ArtemisModuleTestCase):
+    karton_class = SqlInjectionDetector
+
+    def test_sql_injection_detector(self) -> None:
+        task = Task(
+            {"type": TaskType.SERVICE.value, "service": Service.UNKNOWN.value},
+            payload={"url": "http://test-apache-with-sql-injection-postgres"},
+        )
+
+        self.run_task(task)
+        (call,) = self.mock_db.save_task_result.call_args_list
+
+        self.assertEqual(call.kwargs["status"], TaskStatus.INTERESTING)
```

Comment:
check the message as well. and write a second test for mysql whether the message is correct